tests/installation/leap2sle_zdup.pm

```diff
@@ -34,7 +34,8 @@ sub run {
 
     systemctl 'set-default --force multi-user.target';
     fully_patch_system();
-    zypper_call 'in SUSEConnect';
+    # SUSEConnect is obsoleted and provided by suseconnect-ng from 15.4
+    zypper_call 'in SUSEConnect' if is_leap("<=15.3");
```

Comment:
This should still be necessary on 15.4. Is `SUSEConnect` already installed by something else?